doc/Type/List.pod6

```diff
@@ -919,25 +935,49 @@ for operators which aren't left-associative:
     ((2-3)-4).say;                        # OUTPUT: «-5␤»
     (2-3-4).say;                          # OUTPUT: «-5␤»
 
-Since reducing with an infix operator is a common thing to do, the C<[ ]>
-metaoperator provides a syntactic shortcut:
+Since reducing with an infix operator is a common thing to do, the
+L<reduction metaoperator|/language/operators#Reduction_metaoperators> C<[ ]>
+provides a syntactic shortcut. Thus, instead of passing the operator's code
+object to C<reduce>, just pass the operator directly to C<[ ]>. To use a
+user-defined subroutine instead, provide an additional layer of square brackets
+around the subroutine's code object:
 
-    # The following all do the same thing...
-    my @numbers = (1,2,3,4,5);
+    say [*] [1, 2, 3, 4];       # OUTPUT: «24␤»
+    say [min] [4, 2, 1, 3];     # OUTPUT: «1␤»
+
+    sub mult { $^a * $^b };
+    say [[&mult]] [1, 2, 3, 4]; # OUTPUT: «24␤»
```

Comment:
Explain why you're using double hyper here...

Reply:
I mention it in [the paragraph preceding the example](https://github.com/Raku/doc/pull/3447/commits/9ef5e5e452bd163871d35db986c8fc51ccb069d2#diff-cd75d8f66462222cb39d1d41c2823e9cR951-R953).